.github/workflows/test-suite.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10.0-beta.2"]
```

Comment:
Can we just use `"3.10"` here?

Why? / Why not?

Prompted by noticing [the same in a Trio pull request](https://github.com/python-trio/trio/pull/1921/files). Also, useful for context here... where does GitHub document the supported values for `python-version`?

Reply:
Note the extra bit in trio.

https://github.com/python-trio/trio/pull/1921/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR44-R52
```yaml
          # This allows the matrix to specify just the major.minor version while still
          # expanding it to get the latest patch version including alpha releases.
          # This avoids the need to update for each new alpha, beta, release candidate,
          # and then finally an actual release version.  actions/setup-python doesn't
          # support this for PyPy presently so we get no help there.
          # 
          # CPython -> 3.9.0-alpha - 3.9.X
          # PyPy    -> pypy-3.7
          python-version: ${{ fromJSON(format('["{0}", "{1}"]', format('{0}.0-alpha - {0}.X', matrix.python), matrix.python))[startsWith(matrix.python, 'pypy')] }}
```

Reply:
Also https://github.com/actions/setup-python#available-versions-of-python and if you search for `range` you can see it show up in a couple comments.  All around not entirely explicitly documented, but useful.

Reply:
Should I follow `trio`s lead and use the same logic?

Reply:
> Should I follow trios lead and use the same logic?

Perhaps, perhaps not, but either way let's not consider it as part of this pull request.